src/dao.py

```diff
@@ -0,0 +1,82 @@
+import time
+from db import session, ChatStatus, User, Chat, BagelInstance
+
+
+def print_thick_spacer():
+    print('====================')
+
+def print_thin_spacer():
+    print('--------------------')
+
+
+# User
+
+
+def user_print_all(): 
+    print_thick_spacer()
+    print('Users')
+
+    users = User.query().all()
+    for user in users:
+        print(user.name)
+
+    print_thin_spacer()
+
+
+def user_fetch(slack_id, name):
+    user = User.query().filter_by(slack_id=slack_id).first()
+    if user:
+        return user
+    else:
+        user = User(slack_id, name, True)
+        session.add(user) 
+        session.commit()
+        return user
```

Comment:
mmmm not a fan of creating new users on fetch if they don't exist. Could we at least abstract the else block to its own method for adding users and then call it here??